src/main/resources/translator.properties

```diff
@@ -239,19 +239,21 @@ credentialType = type
 credentialType.type = PARAMETER
 
 hudson.model.ChoiceParameterDefinition = choiceParam
-hudson.model.ChoiceParameterDefinition.type = com.adq.jenkins.xmljobtodsl.dsl.strategies.custom.DSLChoiceParamStrategy
+hudson.model.ChoiceParameterDefinition.type = OBJECT
 
-choices = INNER
-choices.type = INNER
+choices = choices
 
 a = a
 a.type = ARRAY
 
 string = string
 string.type = PARAMETER
 
+pre = pre
+pre.type = PARAMETER
+
 hudson.model.BooleanParameterDefinition = booleanParam
-hudson.model.BooleanParameterDefinition.type = com.adq.jenkins.xmljobtodsl.dsl.strategies.custom.DSLParamStrategy
```

Comment:
Same question from your previous PR -- why were we using the custom type before? Was it doing anything we need to pay attention to?

Reply:
In the [Jenkins api viewer](https://jenkins.tinyspeck.com/plugin/job-dsl/api-viewer/index.html#method/javaposse.jobdsl.dsl.helpers.BuildParametersContext.choiceParam) the properties for choiceParam, booleanParam and stringParam are formatted as parameters so that's what the custom class is meant to do. 

<img width="687" alt="Screen Shot 2022-10-25 at 4 49 18 PM" src="https://user-images.githubusercontent.com/112515811/197879056-79ccfcab-3840-4cc6-b59d-3de0d0521c92.png">
<img width="710" alt="Screen Shot 2022-10-25 at 4 50 00 PM" src="https://user-images.githubusercontent.com/112515811/197879181-d70d9a17-7dcb-4f04-a410-5556958b233f.png">

In the [Tinyspeck api viewer](https://jenkins.tinyspeck.com/plugin/job-dsl/api-viewer/index.html#method/javaposse.jobdsl.dsl.jobs.FreeStyleJob.parameters) they are objects. 

<img width="370" alt="Screen Shot 2022-10-25 at 4 51 53 PM" src="https://user-images.githubusercontent.com/112515811/197879530-e14b8b7a-79bb-4d8c-acb9-e15aad1a0476.png">

Even with the custom classes in place, nothing is displaying when the project is run. I'll investigate further to try and find out exactly what is causing that bug.

But looking at these differences between the TS api and the Jenkins should be stick with the formatting on the TS site?